src/plugin/base-plugin.js

```diff
@@ -37,6 +37,11 @@ export default class BasePlugin implements IPlugin {
    * @member
    */
   eventManager: EventManager;
+  /**
+   * Weather it is disabled or not
+   * @member
+   */
+  static _disabled: boolean = false;
```

Comment:
why do we need to save the disabled property in the base plugin?
Should we just not load the plugin if it configured with `disabled: true`?